src/thrift/service.thrift

```diff
@@ -25,4 +25,11 @@ service FmuService {
     defs.Status write_string(1: defs.ValueReferences vr, 2: defs.StringArray value) throws (1: defs.NoSuchInstanceException ex1, 2: defs.NoSuchVariableException ex2)
     defs.Status write_boolean(1: defs.ValueReferences vr, 2: defs.BooleanArray value) throws (1: defs.NoSuchInstanceException ex1, 2: defs.NoSuchVariableException ex2)
 
+    defs.StateIndex save_state() throws (1: defs.NoSuchInstanceException ex1, 2: defs.NoSuchVariableException ex2)
```

Comment:
`NoSuchInstanceException` is not applicable here. `UnsupportedOperation` (needs adding) might be suitable

Reply:
Updated